roles/etcd/templates/etcd-docker.service.j2

```diff
@@ -6,7 +6,7 @@ After=docker.service
 [Service]
 User=root
 PermissionsStartOnly=true
-ExecStart={{ docker_bin_dir | default("/usr/bin") }}/docker run --restart=always \
+ExecStart={{ docker_bin_dir | default("/usr/bin") }}/docker run --restart=on-failure:5 \
```

Comment:
There is `Restart: always` below, seems overriding that param at the systemd service unit layer. Are you sure this change works as expected?